src/helpers/getAdvices.ts

```diff
@@ -1,11 +1,16 @@
-export const getAdvices = async () => {
+import { Advice, AdviceData } from '../types/index.ts';
+
+export const getAdvices = async (): Promise<Advice> => {
   const url = 'https://api.adviceslip.com/advice';
 
   const response = await fetch(url);
-  const data = await response.json();
+  const data:AdviceData = await response.json();
```

Comment:
Si tomas el consejo de renombrar AdviceData a AdviceResponseBody tendrías que renombrarlo aquí también:

```suggestion
  const data: AdviceResponseBody = await response.json();
```